op-node/server/http.go

```diff
@@ -0,0 +1,20 @@
+package server
+
+import (
+	"net/http"
+
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+// Use default timeouts from Geth as battle tested default values
+var timeouts = rpc.DefaultHTTPTimeouts
+
+func NewHttpServer(handler http.Handler) *http.Server {
```

Comment:
I like idea of having a single method that gives a default http.Server instance, including the timeouts. It avoids duplicating code in a bunch of places and if there are other settings that should be applied everywhere in the future we now have a single point for it.

However, I dislike `server` as a module name but `http` conflicts with `net.http` far too much and is worth avoiding given we accept a `http.Handler` as a param. Better suggestions welcome. :)

Reply:
There is a `crypto` package which conflicts with geth's crypto package and we usually import it as `op_crypto` so I'm fine with an `http` package that we import as `op_http`

Reply:
Ok that works. Renamed back to `http` with the import named `ophttp` (we seem to import crypto as `opcrypto` and avoid snake case).